ci/ansible/roles/deploy/templates/gunicorn.py.j2

```diff
@@ -22,7 +21,7 @@ raw_env = [
     'REDIS_URI={{ redis_uri }}',
     'REDIS_URI_RO={{ redis_uri_ro }}'
 ]
-accesslog = '-'
-access_log_format = '"%({X-Forwarded-For}i)s" %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"'
+#accesslog = '-'
```

Comment:
It floods the journal.....a lot.  Left it there so it is easy to uncomment if we need to debug at that level, otherwise all domlogs are sent to the rsyslog server by nginx.  This was senseless extra overhead not needed in prod.